contracts/staking/contracts/src/libs/LibStakingRichErrors.sol

```diff
@@ -443,4 +447,19 @@ library LibStakingRichErrors {
             poolId
         );
     }
+
+    function ZrxWithdrawalFailedError(
```

Comment:
I wonder if it would make sense to include an enum that gives info about why a withdrawal failed in this error. For example, if the `WETH` can't be unwrapped for some reason, it might be useful to include that in this error message.

On the other hand, if there aren't several cases in which withdrawing can fail that aren't like this scenario (which is pretty unlikely), then it's fine to skip this.

Reply:
It's just a `ZRX.transfer` call, so it can really only fail if the balance is insufficient. It actually _should_ never fail, but it felt dirty not checking the return value.